package.json

```diff
@@ -9,6 +9,8 @@
     "type": "git",
     "url": "git://github.com/wikimedia/grunt-banana-checker.git"
   },
+  "homepage": "https://github.com/wikimedia/grunt-banana-checker",
+  "issues": "https://github.com/wikimedia/grunt-banana-checker/issues",
```

Comment:
Hm.. I haven't seen the `issues` key before. Was this meant to be `bugs`? <https://docs.npmjs.com/files/package.json#bugs>

Reply:
Hmm. I took this from `eslint-config-wikimedia` as a field we should have, but yeah, it looks to be wrong.